doc/source/index.rst

```diff
@@ -16,7 +16,8 @@
    :maxdepth: 5
    :numbered:
    :hidden:
-  
+
+   Coding conventions <coding_conventions>
```

Comment:
I think that the code conventions should be place after architecture

Reply:
Finnally placed at the end.

Reply:
I don't see the modification. It is pushed ?

Reply:
Now pushed :-)